source/val/validate_memory.cpp

```diff
@@ -196,29 +196,30 @@ bool ContainsInvalidBool(ValidationState_t& _, const Instruction* storage,
   return false;
 }
 
-bool ContainsCooperativeMatrix(ValidationState_t& _,
-                               const Instruction* storage) {
+bool ContainsType(ValidationState_t& _, const Instruction* storage,
```

Comment:
ValidationState_t has a ContainsType method already that basically does this. Maybe this can be replaced with that?

Reply:
ok, done.